io.catenax.shared.recycling_strategy_certificate/RELEASE_NOTES.md

```diff
@@ -3,7 +3,7 @@ All notable changes to this model will be documented in this file.
 
 ## [Unreleased]
 
-## [3.0.0] - 2024-03-04
```

Comment:
you don't need to change date here for recycling strategy certificate.
This date is always the date when you get MS3 approval. Since this model is already MS3 approved and released, no changes required in this model.